node/rpc.go

```diff
@@ -51,7 +52,8 @@ func ServeRPC(h http.Handler, id string, addr multiaddr.Multiaddr) (StopFunc, er
 
 	// Instantiate the server and start listening.
 	srv := &http.Server{
-		Handler: h,
+		Handler:           h,
+		ReadHeaderTimeout: 3 * time.Second,
```

Comment:
How did we get this number? What was the previous default?

I imagine under high load setting this to just 3 seconds may cause issues

Reply:
I commented above, I just chose this number at random. I'm happy for thoughts on what number to use here and for the other timeouts. Before we didn't have any timeout at all, which is probably bad and why the linter was mad.

Reply:
Let's set this to 30 seconds for now, 3 seconds is almost definitely too low